src/index_runner/es_indexer.py

```diff
@@ -189,7 +202,7 @@ def _write_to_elastic(data):
     json_body = ''
     while data:
         datum = data.pop()
-        idx = f"{_PREFIX}.{datum['index']}"
+        idx = _get_index_name(datum['index'])
```

Comment:
not convinced by this. This will default to the standard prefix for the index_runner instance, which may not be the behavior we want.

Reply:
I imagine that if we have the standard `prefix=WS` and want to define an index with a different prefix like `narrativeci.narrative` for example, we'd want the indexer for narratives to write to `narrativeci.narrative`. As it exists now it would write to `WS.narrative`
One way around this would be to define the 'index' field in the indexer with the prefix, but that presupposes that the Prefix should be consistent across deployments, which creates a separate problem.
To fix this, we would have to have some sort of mapping scheme in the configuration or at least some consistent method of mapping to the appropriate index in the index_runner